website/blog/2024-01-26-Custom-Models/index.mdx

```diff
@@ -0,0 +1,156 @@
+---
+title: "AutoGen with Custom Models: Empowering Users to Use Their Own Inference Mechanism"
```

Comment:
Should we add "Local" into the title? So, it can be distinguished with other types of remote hosted models.

Reply:
Since users can plugin whatever class implementation they want, they could start adding calls to remote hosted models too, so reluctant to add the name "local"